xwiki-platform-core/xwiki-platform-livedata/xwiki-platform-livedata-rest/src/main/java/org/xwiki/livedata/rest/LiveDataEntryResource.java

```diff
@@ -70,7 +70,7 @@ Entry getEntry(
      * @return the response
      * @throws Exception if updating the specified entry fails
      */
-    @PUT
+    @POST
```

Comment:
PUT is for creating a new resource, POST to update an existing one.

Reply:
On the contrary, I think it's the other way around in the scope of REST resources. See for instance https://restfulapi.net/rest-put-vs-post/ . I've used ``PUT`` precisely because of that:
* you create new resources with ``POST`` from the parent resource (POST request to ``/entries``)
* you update an existing resource with ``PUT`` targeting the resource itself (PUT to ``/entry/{entryId}``)